arches/app/models/resource.py

```diff
@@ -544,7 +553,8 @@ def get_documents_to_index(
 
         document["tiles"] = tiles
         document["permissions"] = {
-            "creator": [self.get_instance_creator()["creatorid"]]
+            "creator": [self.get_instance_creator()["creatorid"]],
```

Comment:
Based on the call last week, this does seem different to the principal_user case, so I can definitely see the value in having both - but one question, does searching the edit-log table add overhead to get_documents_to_index? (in fact, that is one of the main reasons I had not gone the "userid-on-table" direction, as we are also trying to trim reindexing time)

Reply:
@philtweir - We're actually all in on the principal user idea. Our intention was to remove the creator in favor of the principal_user just so that we don't have to do that edit_log query anymore. A small performance win - but I'm sure helpful when processing large volumes of resources. Now that Aaron has merged this PR, he's going to go ahead and remove the creator. It looks like a pretty minor change.